Base/usr/share/man/man7/kernel_cmdline.md

```diff
@@ -0,0 +1,129 @@
+For that, you can modify the GRUB configuration files. Since we have 3 of them
+(one for each partition scheme), you should pick the right one for you, so if
+you build a virtual disk with a GPT partition scheme, you should edit
+`grub-gpt.cfg` file.
+
+## Kernel boot parameters list
+
+Since it's pretty much not convenient for beginners to have a travel in the
+`init.cpp` file, the list here is meant to aid users to find all the kernel boot
+parameters and their meaning.
+
+### `text_debug`
+
+This boot parameter requires no value after it. This boot parameter enables
+Shell mode during boot, therefore you won't boot into the graphical mode.
+
+### `no_vmmouse`
+
+This boot parameter requires no value after it. This boot parameter disables the
+VMMouse feature. This boot parameter is useless on real hardware or emulators
+that don't support the VMMouse feature.
+
+### `noacpi`
+
+This boot parameter requires no value after it. This boot parameter disables
+ACPI entirely. Therefore, the OS won't be able to use power management features,
+nor to detect hardware that is only detectable with ACPI.
+
+### `noacpi_aml`
+
+This boot parameter requires no value after it. This boot parameter disables
+ACPI to some extent. Therefore, the OS won't be able to use power management
+features, nor to detect hardware that is only detectable with ACPI. However,
+since this boot parameter only disables AML interpretation, it is still possible
+to use some of ACPI features.
+
+### `dmi_unreliable`
+
+This boot parameter requires no value after it. This boot parameter warns the OS
+that the DMI (SMBIOS) implementation on the machine is not reliable and the DMI
+decoder should be more careful when interpreting the SMBIOS tables. For now this
+boot parameter has no real effect, but in the future it can have.
+
+### `force_pio`
+
+This boot parameter requires no value after it. This boot parameter disables the
+DMA mechanism for the IDE controller. It is meant that that this boot parameter
+will be used for debugging, when one suspects that the IDE controller doesn't
+configure the DMA mechanism like we expect it should.
+
+### `root`
+
+This boot parameter requires value after it. This boot parameter is specified to
+make sure the kernel will mount the right hard drive and the right partition for
+it.
+
+On GPT or MBR schemes that have Extended partitions, there's no theoretical
+limit on partition number, thus, you can specify the parameter like so:
+
+```
+root=/dev/hda5 param2 ...
+```
+
+to boot the first hard drive available by the IDE controller, partition 5.
+However, on pure MBR partition scheme (no DOS extended partitions are present),
+you may only specify a partition number between 1 to 4.
+
+Please note, that DOS extended partitions are limited by the kernel to 128
+partitions.The odds of encountering such configuration are slim anyway, so you
+shouldn't worry about it. GPT partition scheme enforces a limit of about 4
```

Comment:
Uhh, I'm still trying to say this in different ways: "you shouldn't worry about it" is perfect for a "helpful guide" kind of document but not for a man page. "such configurations are extremely rare in practice" (or something) would be more like it.

Reply:
Got it, I'm just unfamiliar with such documents, as you can conclude :)